tsconfig.base.json

```diff
@@ -37,7 +37,7 @@
     "packages/**/dist",
     "packages/**/lib",
     "packages/**/lib-esm",
-    "packages/adapters",
+    "packages/adapters/**/*.js",
```

Comment:
I had to do this because of a new error that started appearing after the latest sync with main:
```
Parsing error: ESLint was configured to run on `<tsconfigRootDir>/packages/adapters/src/adapters/Cornerstone3D/EllipticalROI.ts` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.json
```